site/src/components/screens/send-login-code-screen.tsx

```diff
@@ -27,14 +28,14 @@ type SendLoginCodeScreenProps = {
 
 export const SendLoginCodeScreen: FunctionComponent<
   SendLoginCodeScreenProps
-> = ({ initialEmail, onLoginCodeSent, onClose }) => {
+> = ({ disabled, initialEmail, onLoginCodeSent, onClose }) => {
   const emailInputRef = useRef<HTMLInputElement>(null);
 
   useEffect(() => {
-    if (emailInputRef.current) {
+    if (emailInputRef.current && !disabled) {
       emailInputRef.current.select();
```

Comment:
```suggestion
      emailInputRef.current.focus();
```

Tested and still noticed the same behaviour on safari. I think the problem is with `select`. When I switched to using `focus`, I didn't experience the behaviour

Reply:
It also worked without the `disabled` fix when I tested

Reply:
Does this `useEffect` fire when you press close? It was only seeing it on modal open. Changing to `emailInputRef.current.focus()` did not work for me locally when I tried various solutions yesterday.

Reply:
https://github.com/blockprotocol/blockprotocol/pull/463/commits/8ceac91650f48f1e0469848c6c199cc5effcbf01